app/controllers/users_controller.rb

```diff
@@ -142,11 +142,15 @@ def index
         retired = Group.find_by(name: 'retired')
         rid = retired ? retired.id : -1
 
+        # will_Paginate
         @users = User.includes(memberships: :group)
                      .reject { |u| u.member? rid }
                      .sort { |a, b| a[:login] <=> b[:login] }
                      .paginate(page: params[:page], per_page: 15)
 
+        # alphabetical_Paginate
+        @Users, @alphaParams = User.all.alpha_paginate(params[:letter], {db_mode: true, db_field: "name"})
```

Comment:
Use `@users` instead of uppercase.
Replace `User.all` in this query with `User.includes(memberships: :group).reject {|u| u.member? rid }` from above